src/controller-migration.ts

```diff
@@ -27,7 +22,7 @@ export default function (
 
   return async function (fastify: FastifyInstance) {
     fastify.register(fastifyJwt, { secret: config.JWT_SECRET });
-    fastify.register(rateLimit, {
+    fastify.register(import('@fastify/rate-limit'), {
```

Comment:
Import this one as we do with other libraries, doing this things unless you want to use dynamic imports does not add any advantage